src/views/pages/EventInfo.vue

```diff
@@ -0,0 +1,264 @@
+        </form>
+      </div>
+      <img class="max-w-10 h-auto mt-4 lg:mt-0" :src="event.banner" :alt="event.title" />
+    </div>
+    <div class="grid grid-cols-1 lg:grid-cols-3 gap-x-5 w-full">
+      <div>
+        <h3 class="mb-1">Enroute Positions</h3>
+        <Positions type="Enroute" :positions="event.positions" :signups="event.signups" :event-id="id" />
+      </div>
+      <div>
+        <h3 class="mb-1">TRACON Positions</h3>
+        <Positions type="TRACON" :positions="event.positions" :signups="event.signups" :event-id="id" />
+      </div>
+      <div>
+        <h3 class="mb-1">Local Positions</h3>
+        <Positions type="Local" :positions="event.positions" :signups="event.signups" :event-id="id" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { hasRole, isAuthenticated } from "@/utils/auth";
+import { onMounted, ref } from "vue";
+import { useRoute, useRouter } from "vue-router";
+
+import CreatePositionModal from "@/views/partials/events/CreatePositionModal.vue";
+import Positions from "@/views/partials/events/Positions.vue";
+import SignupModal from "@/views/partials/events/SignupModal.vue";
+
+import { Event } from "@/types";
+
+import { useDark } from "@vueuse/core";
+import useEventStore from "@/stores/event";
+import { ZDVAPI } from "@/utils/api";
+
+const isDark = useDark();
+const loading = ref(true);
+const route = useRoute();
+const router = useRouter();
+const eventStore = useEventStore();
+const id = parseInt(route.params.id as string, 10);
+const event = ref(eventStore.getEvent(id) as Event);
+const modifiedEvent = ref({} as Event);
+const editing = ref(false);
+
+const localDate = (s: string): string => {
+  const dt = new Date(s);
+  return `${dt.getMonth() + 1}/${dt.getDate()}/${dt.getFullYear()}`;
+};
+
+const canEditEvent = (): boolean => {
+  return isAuthenticated() && hasRole(["atm", "datm", "ta", "ec"]);
+};
+
+const canDeleteEvent = (): boolean => {
+  return isAuthenticated() && hasRole(["atm", "datm", "ta", "ec"]);
+};
+
+const canSignup = (): boolean => {
+  return isAuthenticated();
+};
+
+enum ButtonStates {
+  Idle = 0,
+  Saving = 1,
+  Error = 2,
+}
+
+const buttonState = ref(ButtonStates.Idle);
+
+const editEvent = (): void => {
+  if (canEditEvent()) {
+    buttonState.value = ButtonStates.Idle;
+    modifiedEvent.value.title = event.value.title;
+    modifiedEvent.value.description = event.value.description;
+    modifiedEvent.value.start_date = event.value.start_date;
+    modifiedEvent.value.end_date = event.value.end_date;
+    modifiedEvent.value.banner = event.value.banner;
+    editing.value = true;
+  }
+};
+
+const deleteEvent = async (): Promise<void> => {
+  if (canEditEvent()) {
+    try {
+      const result = await ZDVAPI.delete(`/v1/events/${id}`);
+      if (result.status === 204) {
+        await eventStore.fetchEvents();
+        await router.push(`/events/`);
+      }
+    } catch (error) {
+      console.error(error);
```

Comment:
Something should happen here so the user knows the delete failed. Look at the Alert component.

Reply:
Fixed in [b401cbc]